pkg/storaged/client.js

```diff
@@ -1774,9 +1774,9 @@ client.export_mount_point_mapping = () => {
                 const type = utils.decode_filename(c[1].type.v);
 
                 if (dir)
-                    mpm[dir] = device;
+                    mpm[device] = dir;
```

Comment:
I would still prefer `mpm[device] = { dir, type }`. Then we have a structure that can easily be extended.

Reply:
Agreed :)